scss/styles.scss

```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+body {
```

Comment:
use `*` instead, you dont want this style in body, you want this to reach every kind of html tag